test/topology/test_topology.py

```diff
@@ -154,3 +155,14 @@ async def do_remove_node():
         stopped = True
         await ddl_task
         logger.debug("ddl fiber done, finished")
+
+
+@pytest.mark.asyncio
+@pytest.mark.xfail(reason="group0 remove exception is not properly handled")
```

Comment:
I usually mention the issue number (in this case #11723) in the xfail message so it's easy to search for tests reproducing some issue. I also like to mention the issue number *again* in the test's comment, e.g., "Reproduces #11723" - this string will remain even after the issue is solved, and make it easy to understand which issue came back (or wasn't backported!) if we see see this test failing later.